src/choose-location/choose-location.component.tsx

```diff
@@ -68,7 +68,7 @@ export default function ChooseLocation(props: ChooseLocationProps) {
   return (
     <div className={`canvas ${styles["container"]}`}>
       <h1 className={styles["welcome-msg"]}>
-        <Trans i18nKey="welcome">Welcome </Trans>
+        <Trans i18nKey="welcome">Welcome</Trans>&nbsp;
```

Comment:
Should it be non-breaking? Would `{' '}` be sufficient?

Reply:
My personal reason for breaking space: I am not sure if long names should not be put on the second line on mobile.

Having

```
Welcome
VERYLONGNAMEONMOBILE
```

could be more appealing than

```
Welcome VERYLONGN
AMEONMOBILE
```

or other options (ellipsis, horizontal scrollbar etc.).

Thoughts?